Description: fix ACE_DYNAMIC_VERSIONED_SERVICE_DIRECTIVE
 Because of '.' in the version, the automatic
 addition of prefix "lib" and suffix ".so" is
 skipped, preventing ACE_DYNAMIC_VERSIONED_SERVICE_DIRECTIVE
 from working when versioned_so is set to 2.
 
 Fix this by explicitely adding ACE_DLL_PREFIX and ACE_DLL_SUFFIX
 in the macro.
Forwarded: yes
Author: Thomas Girard <thomas.g.girard@free.fr>
Last-Update: 2013-01-01

--- pkg-ace.orig/ace/Service_Config.h
+++ pkg-ace/ace/Service_Config.h
@@ -62,9 +62,11 @@
   ACE_TEXT ("dynamic ") \
   ACE_TEXT (ident) \
   ACE_TEXT (" Service_Object * ") \
+  ACE_DLL_PREFIX \
   ACE_TEXT (libpathname) \
   ACE_TEXT ("-") \
   ACE_TEXT (version) \
+  ACE_DLL_SUFFIX \
   ACE_TEXT (":") \
   ACE_TEXT (objectclass) \
   ACE_TEXT ("() \"") \
